Add teaser to online publication

curzio-della-santa opened the pull request.

curzio-della-santa committed:
Prototype of online-publication assortment-article

curzio-della-santa committed:
Prototype v2 of online-publication assortment-article

curzio-della-santa committed:
Fix typo

curzio-della-santa committed:
Removed unneeded

curzio-della-santa committed:
Change example name

curzio-della-santa committed:
Refactor to single product-line, only basic data

curzio-della-santa committed:
Multi language stopper text

curzio-della-santa committed:
Fix stopper text property

ivan-wolf commented on examples/online-publication-with-product-lines.json:
```diff
@@ -0,0 +1,1376 @@
+        "id": "promotion",
+        "name": "Aktionsmix"
+    },
+    "product_line": {
```
```suggestion
    "teaser": {
```

ivan-wolf commented on examples/online-publication-with-product-lines.json:
```diff
@@ -0,0 +1,1376 @@
+                "value": "..."
+            }
+        ],
+        "stopper_text": [
```
```suggestion
        "stopper": [
```

ivan-wolf commented on examples/online-publication-with-product-lines.json:
```diff
@@ -0,0 +1,1376 @@
+            "id": "104cc3ae-faec-4f0f-ac18-8e6d3e8e1dd1",
+            "cdn_url": "https://denner.imgix.net/assets/104cc3ae-faec-4f0f-ac18-8e6d3e8e1dd1/web/051683_6.png"
+        },
+        "packshot_production_note": "<span style=\"background-color: rgb(247, 247, 247);\">\u200BFolgende Artikel verwenden:</span><ul style=\"\"><li style=\"background-color: rgb(247, 247, 247);\">000000</li><li style=\"\"><span style=\"background-color: rgb(255, 255, 0);\">1111111</span></li><li style=\"\"><span style=\"background-color: rgb(255, 255, 0);\">222222</span></li><li style=\"backgro ...
```
```suggestion
        "remarks": "<span style=\"background-color: rgb(247, 247, 247);\">\u200BFolgende Artikel in Bild zeigen:</span><ul style=\"\"><li style=\"background-color: rgb(247, 247, 247);\">000000</li><li style=\"\"><span style=\"background-color: rgb(255, 255, 0);\">1111111</span></li><li style=\"\"><span style=\"background-color: rgb(255, 255, 0);\">222222</span></li><li style=\"background-color: rgb(247, 247, 247);\">333333</li></ul>Den mittleren Artikel 30 % grösser darstellen."
```

ivan-wolf commented on examples/online-publication-with-product-lines.json:
```diff
@@ -0,0 +1,1376 @@
+{
```
Why is a separate file needed? Can't we add `teaser` to [examples/online-publications.json](https://github.com/detailnet/denner-portal-api-spec/blob/master/examples/online-publications.json)?

curzio-della-santa replied:
It makes sense.

The reason to make the new one, is because I've taken a real online-publication .. to provide a correct example

I'll integrate all in the same

ivan-wolf replied:
@curzio-della-santa, it would be good to have a PR with only the `teaser` as diff, so that we can send the link to Visual Fantastix (they might get confused when there's a whole lot of unrelated changes).

curzio-della-santa committed:
Include review feedback

curzio-della-santa committed:
Describe OnlineTeaser

Move teaser into online-publcations example.

Swagger definition of teaser into online publication.

curzio-della-santa commented:
![image](https://user-images.githubusercontent.com/7221909/108174846-48c8f400-7100-11eb-9ee4-417108498784.png)

curzio-della-santa committed:
Merge branch 'master' into prototype/online-publication-assortment-article

curzio-della-santa committed:
Add sales and instead price

ivan-wolf reviewed (approved):
@curzio-della-santa, Visual Fantastix agreed to the proposed changes.

curzio-della-santa commented on examples/online-publications.json:
```diff
@@ -669,6 +669,84 @@
+                        "url": "https://www.denner.ch/it/shop/vino/vini.html"
+                    }
+                ],
+                "packshot": {
+                    "id": "104cc3ae-faec-4f0f-ac18-8e6d3e8e1dd1",
+                    "cdn_url": "https://denner.imgix.net/assets/104cc3ae-faec-4f0f-ac18-8e6d3e8e1dd1/web/051683_6.png"
+                },
```
Changed to 
```Json
 "packshot_url":  "https://denner.imgix.net/assets/104cc3ae-faec-4f0f-ac18-8e6d3e8e1dd1/web/051683_6.png"
```
